privacyidea/static/components/token/controllers/tokenControllers.js

```diff
@@ -168,9 +168,15 @@ myApp.controller("tokenEnrollController", function ($scope, TokenFactory,
     });
 
     $scope.qrCodeWidth = 250;
-    if ($state.includes('token.wizard')) {
+    $scope.hide_wizard_seed = true;
+    if ($state.includes('token.wizard') && !$scope.show_seed) {
         $scope.qrCodeWidth = 500;
     }
+
+    if ($state.includes('token.wizard') && $scope.show_seed) {
+        $scope.hide_wizard_seed = false;
+    }
+
     $scope.checkRight = AuthFactory.checkRight;
     $scope.loggedInUser = AuthFactory.getUser();
     $scope.newUser = {};
```

Comment:
we cannot revert it, because we need to change size of QR image in case of token wizard, because token wizard shows 500px image in case there is no otp seed shown